Public/Src/Utilities/Native/Processes/ProcessUtilities.cs

```diff
@@ -287,21 +287,30 @@ public static bool OSSupportsNestedJobs()
         /// The Helium container is configured using the provided path mapping. The mapping is expected to 
         /// contain destination -> [source]. For each entry in the dictionary, two file 
         /// drivers are setup, Wci and Bind:
+        ///
         /// - WCI filter is able to virtualize source folders into destination folders via reparse points: processes
         /// running in a container will see the content of the destination folder as containing the files of the source
         /// folder. Plus, it provides full isolation from the source folder (copy-on-write semantics, tombstone files for deletion)
         /// - Bind filter is able to map a source folder into a target folder, so any process running in a container will get every access
         /// to the source path translated into an access to the target path
+        ///
         /// Working together, these filters can provide full virtualization: a process accessing a source path will be redirected under the hood
         /// to the target path, and the content that it will see there is fully controlled.
         /// </remarks>
         /// <param name="hJob">A pointer to a job object where a specific filter configuration will be associated with. 
         /// This is the result of calling CreateJobObject <see href="https://msdn.microsoft.com/en-us/library/windows/desktop/ms682409(v=vs.85).aspx"/></param>
         /// <param name="redirectedDirectories">The collection of source paths to be virtualize to destination paths</param>
+        /// <param name="isWciFilterEnabled">Enables WCI filter for inpuit virtualization</param>
+        /// <param name="bindFltExclusions">Paths to not apply the bindflt path transformation to.</param>
         /// <param name="warnings">Any warnings that happened during the creation of the container. The container was created successfully regardless of these.</param>
         /// <exception cref="BuildXLException">If any unrecoverable error occurs when setting up the container</exception>
-        public static void AttachContainerToJobObject(IntPtr hJob, IReadOnlyDictionary<ExpandedAbsolutePath, IReadOnlyList<ExpandedAbsolutePath>> redirectedDirectories, out IEnumerable<string> warnings) 
-            => s_nativeMethods.AttachContainerToJobObject(hJob, redirectedDirectories, out warnings);
+        public static void AttachContainerToJobObject(
+            IntPtr hJob,
+            IReadOnlyDictionary<ExpandedAbsolutePath, IReadOnlyList<ExpandedAbsolutePath>> redirectedDirectories,
+            bool isWciFilterEnabled,

```

Comment:
>isWciFilterEnabled [](start = 17, length = 18)

I think a better name would be 'enableWciFilter'.
From what I can see, this flag is for instructing the API to configure WCI. The name makes it look as if we were querying something